bin/php-gameboy

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env php
+<?php
+
+require __DIR__.'/../boot.php';
```

Comment:
Actually I'm favor of moving the content of `boot.php` to this file, but that's your call.